src/compression_wrapper.c

```diff
@@ -26,9 +26,11 @@
 #include <string.h>
 #include <stdio.h>
 #include <ctype.h>
+#include <stdbool.h>
 #include <zlib.h>
 #include <bzlib.h>
 #include <lzma.h>
+#include <zck.h>
```

Comment:
This needs to be wrapped into #ifdef WITH_ZCHUNK

Reply:
Ok, this is done now.  I did a test build on a system without zchunk, and it built and ran just fine.

Reply:
Thanks so much for committing these patches!!!